DAQ/src/StrawAndCaloDigisFromFragments_module.cc

```diff
@@ -452,7 +462,7 @@ void art::StrawAndCaloDigisFromFragments::analyze_calorimeter_(
         uint16_t apdID = hits[hitIdx].first.DIRACB >> 12;
 
         // FIXME: Can we match vector types here?
-        std::vector<int> caloHits(hits[hitIdx].second.size());
+        std::vector<int> caloHits;
         std::copy(hits[hitIdx].second.begin(), hits[hitIdx].second.end(), std::back_inserter(caloHits));
```

Comment:
Does the copy does an implicit reserve or do we need to do that ourselves?  I bet it does but I don't know.  Please check.   I agree that you don't need to default construct everything in the vector.

Reply:
Thanks for checking this. `std::copy` doesn't call any reserve internally. I'll add it now.